### ISSUE TITLE
The scrollTo command doesn't log coordinates when options are used

### ISSUE THREAD

**Issue by verheyenkoen** [cypress-io/cypress#725]
- Operating System: Mac OS X
- Cypress Version: 1.0.1
- Browser Version: Chrome 61

### Is this a Feature or Bug?
Bug

### Current behavior:
```
cy.scrollTo(0, 200, {duration: 500});
```
In command log:
![image](https://user-images.githubusercontent.com/599538/31449769-1d01b87a-aea8-11e7-9082-bd0bdc9da078.png)

In debug console:
![image](https://user-images.githubusercontent.com/599538/31449787-2a07df86-aea8-11e7-82e2-5204b0931be9.png)

### Desired behavior:
In command log:
```
SCROLLTO 0, 200, {duration: 500}
```

In debug console:
```
Command: scrollTo
X: 0
Y: 200
Options: {duration: 500}
```
#### Test code:

```js
cy.visit('http://google.com');

cy.scrollTo(0, 200, {duration: 500});
```

**Comment by sirugh** [comment]
I can look into this one!

**Comment by brian-mann** [comment]
The code for this is merged, but it has not been released yet.

**Comment by brian-mann** [comment]
Fixed in [`1.0.2`](https://docs.cypress.io/guides/references/changelog.html#1-0-2).
